src/main/java/com/keboola/tableexporter/Application.java

```diff
@@ -23,14 +18,14 @@
     private static String outputFile;
     private static Connection connection;
     
-    private static void readConfigFile() throws ApplicationException {
-        System.out.println("Processing configuration file");
+    private static void readConfigFile(String configFile) throws ApplicationException {
+        System.out.println("Processing configuration file " + configFile);
         String jsonString;
         try {
-            String configPath = System.getenv("KBC_DATADIR") + "/config.json";
             byte[] encoded;

```

Comment:
isn't it better to keep the env variable here? it's usually useful in tests to be able to specify different config files